docs/install_guides/installing-openvino-overview.md

```diff
@@ -13,46 +13,57 @@
 
 @endsphinxdirective
 
-Intel® Distribution of OpenVINO™ toolkit is a comprehensive toolkit for developing applications and solutions based on deep learning tasks, such as: emulation of human vision, automatic speech recognition, natural language processing, recommendation systems, etc. It provides high-performance and rich deployment options, from edge to cloud. Some of its advantages are:
+Intel® Distribution of OpenVINO™ toolkit is a comprehensive toolkit for developing applications and solutions based on deep learning tasks, such as emulation of human vision, automatic speech recognition, natural language processing, recommendation systems, and more. It provides high-performance and rich deployment options, from edge to cloud. Some of its advantages are:
 
-* Enabling CNN-based deep learning inference on the edge.
-* Supporting various execution modes across Intel® technologies: Intel® CPU, Intel® Integrated Graphics, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs.
-* Speeding time-to-market via an easy-to-use library of computer vision functions and pre-optimized kernels.
+* Enables CNN-based and transformer-based deep learning inference on the edge or cloud.
+* Supports various execution modes across Intel® technologies: Intel® CPU, Intel® Integrated Graphics, Intel® Discrete Graphics, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs.
+* Speeds time-to-market via an easy-to-use library of computer vision functions and pre-optimized kernels.
+* Compatible with models from a wide variety of frameworks, including TensorFlow, PyTorch, PaddlePaddle, ONNX, and more.
 
-## Installation Options
+## Install OpenVINO
 
-Since the 2022.1 release, the OpenVINO installation package has been distributed in two parts: OpenVINO Runtime and OpenVINO Development Tools. See the following instructions to choose your installation process.
+The OpenVINO installation package is distributed in two parts: OpenVINO Runtime and OpenVINO Development Tools.
 
-### Decide What to Install
+* **OpenVINO Runtime** contains the core set of libraries for running machine learning model inference on processor devices.
+* **OpenVINO Development Tools** is a set of utilities for working with OpenVINO and OpenVINO models. It includes the following tools:
+  - Model Optimizer
+  - Post-Training Optimization Tool
+  - Benchmark Tool
+  - Accuracy Checker and Annotation Converter
+  - Model Downloader and other Open Model Zoo tools
 
-**If you have already finished developing your models and converting them to the OpenVINO model format, you can [install OpenVINO Runtime](installing-openvino-runtime.md) to deploy your applications on various devices**. OpenVINO Runtime contains a set of libraries for easy inference integration with your products.
+### Option 1. Install OpenVINO Development Tools (recommended)
```

Comment:
Sorry I forgot to add this comment in the draft:
so according to the definitions above, dev tools does not contain Runtime, but this would actually install them both. We know that to install dev tools, you must install runtime, but that is not clear to new users in the heading.
So I suggest:
```suggestion
### Option 1. Install OpenVINO Runtime and OpenVINO Development Tools (recommended)
```